Program/stud_fees_data.py

```diff
@@ -0,0 +1,57 @@
+#Program [ 1 ]
+#Create table stud(id, name, dept, paidfee) with 8 records.
+#Create stud.csv file of the stud data (Department will be BBA and BCA).
+#Read csv file using reader object and write data with remaining fee.
+#( Total fee of bba is 13400 and bca is 15400 Rs)
+
+import csv
+
+#Read the CSV file and print data from the file with append()
+#with skipping the header
+
+students = []
+
+with open("D:\\sqlite\\csv\\stud1.csv", "r") as f:

```

Comment:
why you use with statement to open file .

Reply:
> why you use with statement to open file .

I used with statement to open file coz its beneficial that Python automatically closes the file and we don't need to close the file manually such as " file.close() "